src/main/resources/public/js/realmapps.js

```diff
@@ -409,7 +409,8 @@ angular.module('aac.controllers.realmapps', [])
         var slug = $stateParams.realmId;
         var clientId = $stateParams.clientId;
 
-        $scope.isRealmAdmin = $rootScope.user.authorities.some(a => (a.realm === slug && a.role === 'ROLE_ADMIN'));
+        $scope.isAdmin = $rootScope.user.authorities.some(a => (a.authority === 'ROLE_ADMIN'));
+        $scope.isRealmAdmin = $scope.isAdmin || $rootScope.user.authorities.some(a => (a.realm === slug && a.role === 'ROLE_ADMIN'));
         $scope.isRealmDev = $scope.isRealmAdmin || $rootScope.user.authorities.some(a => (a.realm === slug && a.role === 'ROLE_DEVELOPER'));
```

Comment:
Please add `isAdmin` as another sufficent condition for `isRealmDev`, same as `isRealmAdmin`.
SuperAdmin is always a dev in every realm